RELEASE-NOTES.txt

```diff
@@ -1,3 +1,13 @@
+1.25.0
+------
+* New block: Cover
+* Improve icon on the "Take a Video" media option
+* Removed the dimming effect on unselected blocks
+* Enabled edit button over image block for Android
+* [iOS] Add alignment options for heading block
+* Implemented dropdown toolbar for alignment toolbar in Heading, Paragraph, Image, MediaText blocks
+* Block Editor: When editing link settings, tapping the keyboard return button now closes the settings panel as well as closing the keyboard.
+
```

Comment:
Let's add an entry for: https://github.com/wordpress-mobile/WordPress-Android/pull/11557
```suggestion
* Implemented dropdown toolbar for alignment toolbar in Heading, Paragraph, Image, MediaText blocks
* Block Editor: When editing link settings, tapping the keyboard return button now closes the settings panel as well as closing the keyboard.
* [Android] Show an "Edit" button overlay on selected image blocks
```